curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64ec9343769e8f85c1e17e05.md

```diff
@@ -14,17 +14,17 @@ Create a `div` element with the class of `task`. Utilize template strings to set
 You should create a `div` element with the class `task`.
 
 ```js
-const classFirst = `<div\s+class\s*=\s*('|")task\1`;
-const idFirst = `<div\s+id\s*=\s*('|")task\2\s+class\s*=\s*('|")task\3`
-assert.match(code, new Regex(`${classFirst}|${idFirst}`));
+const classFirst =  /<div\s+class\s*=\s*('|")task\1/.test(code);
+const idFirst =  /<div\s+id\s*=\s*('|")\$\{id\}\1\s+class\s*=\s*('|")task\2/.test(code);
+assert.isTrue(classFirst || idFirst);
```

Comment:
To make it work with the way I was suggesting the two strings need to be
```js
const classFirst = `<div\\s+class\\s*=\\s*('|")task\\1`;
const idFirst = `<div\\s+id\\s*=\\s*('|")\\$\\{id\\}\\1\\s+class\\s*=\\s*('|")task\\2``;
```
I got the strings to use using the `source` property of the regexes
Like this it should not give the error that it was giving

not necessary to do the change, just to let you know

Reply:
That's pretty cool. But I don't wanna mess with it if I don't have to.